CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
```

Comment:
You don't have to manually update `CHANGELOG.md` and `pyproject.toml`, it will be done automatically when I'll launch a new release. Or you can add a `RELEASE.md` file [to do it](https://docs.getpelican.com/en/latest/contribute.html).